lib/tasks/resumis.rake

```diff
@@ -1,57 +0,0 @@
-require 'optparse'
-
-namespace :resumis do |t|
-  desc "Task for adding a new, active user to Resumis"
-  task useradd: :environment do
```

Comment:
I'd like to keep this around and just have it call `thor resumis:useradd`, just so folks don't have to remember which things to call in `thor` and which to call in `rake`

Reply:
But with the rake task you have to provide the arguments and with the thor task no.

You want me to allow the thor task to receive arguments so if it has them do not ask for them?

Or simply change the rake task to call the thor task and ask for the arguments again?

Reply:
Just having the rake task call the thor task should be sufficient (ignoring arguments, letting the thor task ask for them)